### ISSUE TITLE
Shake unmaximize other IDE windows

### ISSUE THREAD

**Issue by jonathanlermitage** [hsz/shakeoff#1]
On MS Windows 10. I have two projects opened, in two different IDE windows (same IDE). Both windows are maximized. 
Project 2 has focus. When shaking, Project 1's window is no longer maximized.  
And vice versa: if Project 1 has focus, the problem happens on Project 2.  

Please see attached screen recording: the left and right windows are maximized. I shake in the right window, then I observe the left window is no longer maximized.  
Nota: my IDE windows icons are not combined in the Windows taskbar. I am using _7+ Taskbar Tweaker_ for that. This is the same single IDE instance.

https://github.com/hsz/shakeoff/assets/857535/b2115b9f-2e6c-4d30-afae-5b6d0d1e90c9

**Comment by hsz** [comment]
Thanks for reporting! Do you see any window shaking?
This is how it looks on macOS:

https://github.com/hsz/shakeoff/assets/108333/7e372c64-f587-49c7-8697-e994d9e5302c

Maybe this simply doesn't work on Windows and as a side-effect, window shrinks?

**Comment by jonathanlermitage** [comment]
Yeah, I confirm the other window is shaking. Also, I observe this is always the first window that shakes, even if it has focus. If I shake in window 1, it is un-maximized, but window 2 is not affected (my first comment was not correct, sorry).  
Also tried to de-activate _7+ Taskbar Tweaker_, just to be sure. This changed nothing.

https://github.com/hsz/shakeoff/assets/857535/ba1eded8-3d15-44b1-9c13-f2cbc688ff6a

**Comment by gt-fm-4562** [comment]
> Thanks for reporting! Do you see any window shaking? This is how it looks on macOS:
> Screen.Recording.2023-08-09.at.9.42.38.PM.mov
> 
> Maybe this simply doesn't work on Windows and as a side-effect, window shrinks?

I have exactly the same issue on:
 - Fedora 38 KDE Spin
 - PyCharm 2023.2.1 (Professional Edition) - PY-232.9559.58

In short, shaking the mouse(horizontally) on any PyCharm window, dialogue, etc. causes shaking of the window itself
